pom.xml

```diff
@@ -27,17 +27,17 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>23.0</version>
+      <version>27.0-jre</version>
     </dependency>
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-speech</artifactId>
-      <version>0.49.0-alpha</version>
+      <version>0.68.0-beta</version>
     </dependency>
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-translate</artifactId>
-      <version>1.31.0</version>
+      <version>1.50.0</version>
```

Comment:
The google-cloud-translate API needs to be updated because of dependency issues; not updating it made it impossible to receive results from the Google Cloud server.